parser/src/cfg/mod.rs

```diff
@@ -1111,8 +1111,8 @@ fn parse_action_atom(ac_span: &Spanned<String>, s: &ParsedState) -> Result<&'sta
         );
     }
     match ac {
-        "_" => return Ok(s.a.sref(Action::Trans)),
-        "XX" => return Ok(s.a.sref(Action::NoOp)),
+        "_" | "‗" | "≝" => return Ok(s.a.sref(Action::Trans)),
+        "XX" | "✗" | "∅" | "‿" => return Ok(s.a.sref(Action::NoOp)),
```

Comment:
I think `‿` is too visually similar to `_`, so I would prefer to exclude it from here. The other changes look fine though.

Reply:
Oh, that's actually what I like about it - it's "below" all other symbols just like transparent, so it doesn't take your attention from the regular key definitions, but at the same time not so similar that you can't tell them apart, like check what I used for the fancy symbols example: think it's dissimilar enough to notice

(maybe there is a symbol that's below, but has some "cross" to make it less similar to `_`) in it...

```
 ‗	‗	‗	‗	‗
  ‗	‗	‗	‗	‗	‗	‗	‗	‗	‗	‗
  ‗
  ‗	‗	‗	‗	‗	‗	‗	‗	‗	‗ 	‗	‗	‗	‗	‗	‗	‗	‗	‗	‗	‗
  ‗	‗	‗	‗	‗	‗	‗	‗	‗	‗ 	‗	‗	‗	‗	‗	‗	‗	‗	‗	‗	‗
  ‗	‗	‗	‗	‗	‗	‗	‗	‗	‗ 	‗	‗	‗	 	 	 	 	‗	‗	‗
  ‗	‗	‗	‗	‗	‗	‗	‗	‗	‗ 	‗	‗	 	‗	 	 	 	‗	‗	‗	‗
  ‗	‗	‗	‗	 	 	 	 	 	@🔣	 	‗	‗	‗	‗	‗	 	‗	‗           )
(deflayer  fancy-symbol ;; ‿block all other keys
  🔣‹🖰	🔣🖰›	🔣🖰3	🔣🖰4	🔣🖰5
  🔣▶⏸	🔣◀◀	🔣▶▶	🔣🔇 	🔣🔉	🔣🔊	🔣🔅	🔣🔆	🔣🎛	🔣⌨💡+	🔣⌨💡−
  🔣⎋
  🔣ˋ	  	‿ 	‿ 	‿	‿	‿	‿	‿	‿ 	‿ 	‿ 	🔣‐ 	🔣₌	🔣␈	🔣⎀	🔣⇤	🔣⇞	🔣⇭   	🔣🔢⁄	🔣🔢∗	🔣🔢₋
  🔣⭾	  	‿ 	‿ 	‿	‿	‿	‿	‿	‿ 	‿ 	‿ 	🔣【 	🔣】	🔣⧵	🔣␡	🔣⇥	🔣⇟	🔣🔢₇  	🔣🔢₈	🔣🔢₉	🔣🔢₊
  🔣⇪	  	‿ 	‿ 	‿	‿	‿	‿	‿	‿ 	‿ 	🔣︔	'  	🔣⏎	  	  	  	  	  🔣🔢₄	🔣🔢₅	🔣🔢₆
  🔣⇧	  	‿ 	‿ 	‿	‿	‿	‿	‿	🔣⸴	🔣．	🔣⁄	@⇧🔣	  	🔣▲	  	  	  	  🔣🔢₁	🔣🔢₂	🔣🔢₃	🔣🔢⏎
  🔣⎈	🔣◆	🔣⎇	🔣␠	 	 	 	 	 	‿ 	  	🔣☰	‿  	🔣◀	🔣▼	🔣▶	  	  	  🔣🔢₀	🔣🔢⸴  )
(deflayer ⇧fancy-symbol ;; ‿block all other keys
  🔣🖰1	🔣🖰2	‿	‿    	‿
  ‿  	‿  	‿	🔣🔈⓪⓿₀	‿	🔣🔈−➖₋⊖	🔣🔈+➕₊⊕	‿	‿	🔣⌨💡➕₊⊕	🔣⌨💡➖₋⊖
  ‿
  🔣˜	   	‿ 	‿ 	‿	‿	‿	‿	‿	‿  	‿	‿	-   	=   	🔣⌫	‿ 	🔣⤒↖	🔣🔢	‿	‿	‿	‿
  🔣↹	   	‿ 	‿ 	‿	‿	‿	‿	‿	‿  	‿	‿	🔣「〔⎡	🔣」〕⎣	🔣＼	🔣⌦	🔣⤓↘	‿ 	‿	‿	‿	‿
  ‿ 	   	‿ 	‿ 	‿	‿	‿	‿	‿	‿  	‿	‿	‿   	🔣↩⌤␤	  	  	   	  	 	‿	‿	‿
  ‿ 	   	‿ 	‿ 	‿	‿	‿	‿	‿	‿  	‿	/	‿   	    	‿ 	  	   	  	 	‿	‿	‿	🔣🔢↩⌤␤
  🔣⌃	🔣❖⌘	🔣⌥	🔣␣	 	 	 	 	 	🔣▤𝌆	 	‿	‿   	‿   	‿ 	‿ 	   	  	‿	‿   )
```

Reply:
what do you think about `•`, which ~resembles zero and is smaller than `✗` (it's thick and centered, so no very easy to confuse with a period)

```
(deflayer  fancy-symbol ;; •block all other keys
  🔣‹🖰	🔣🖰›	🔣🖰3	🔣🖰4	🔣🖰5
  🔣▶⏸	🔣◀◀	🔣▶▶	🔣🔇 	🔣🔉	🔣🔊	🔣🔅	🔣🔆	🔣🎛	🔣⌨💡+	🔣⌨💡−
  🔣⎋
  🔣ˋ	  	• 	• 	•	•	•	•	•	• 	• 	• 	🔣‐ 	🔣₌	🔣␈	🔣⎀	🔣⇤	🔣⇞	🔣⇭   	🔣🔢⁄	🔣🔢∗	🔣🔢₋
  🔣⭾	  	• 	• 	•	•	•	•	•	• 	• 	• 	🔣【 	🔣】	🔣⧵	🔣␡	🔣⇥	🔣⇟	🔣🔢₇  	🔣🔢₈	🔣🔢₉	🔣🔢₊
  🔣⇪	  	• 	• 	•	•	•	•	•	• 	• 	🔣︔	'  	🔣⏎	  	  	  	  	  🔣🔢₄	🔣🔢₅	🔣🔢₆
  🔣⇧	  	• 	• 	•	•	•	•	•	🔣⸴	🔣．	🔣⁄	@⇧🔣	  	🔣▲	  	  	  	  🔣🔢₁	🔣🔢₂	🔣🔢₃	🔣🔢⏎
  🔣⎈	🔣◆	🔣⎇	🔣␠	 	 	 	 	 	‿ 	  	🔣☰	‿  	🔣◀	🔣▼	🔣▶	  	  	  🔣🔢₀	🔣🔢⸴  )
```